libr/util/file.c

```diff
@@ -1276,29 +1276,41 @@ R_API bool r_file_copy(const char *src, const char *dst) {
 }
 
 R_API bool r_file_dir_recursive(RList *dst, const char *dir) {
```

Comment:
imho this function should be defined like this:

```R_API RList *r_sys_dir_recursive(const char *path) {```

and return the list instead of bool. we can later decide if dirlisting apis are related to RFile or RSys later.